app/assets/stylesheets/common.scss

```diff
@@ -191,7 +187,7 @@ body.small-nav {
     min-height: $headerHeight;
 
     &.closed nav {
```

Comment:
I wonder: do we use this .closed in a lot of places? Otherwise we could use a utility class on nav to hide it, right?

Reply:
It's used only here. It controls the visibility in response to the menu button. You wouldn't need any class for that, just show/hide from javascript. It was done in #4365.

Reply:
Sounds good. I like that #4365 make it that we stay in one world to toggle the visibility and not scatter logic between JS and CSS (and HTML).

Reply:
I'm not pursuing https://github.com/openstreetmap/openstreetmap-website/pull/4365 right now because maybe I'll go straight to [Bootstrap navbar](https://getbootstrap.com/docs/5.3/components/navbar/) which has its own javascript.